packages/flutter_tools/test/general.shard/web/devfs_web_ddc_modules_test.dart

```diff
@@ -1253,7 +1282,9 @@ void main() {
         isWasm: false,
         useLocalCanvasKit: false,
         rootDirectory: globals.fs.currentDirectory,
-        isWindows: false,
+        fileSystem: globals.fs,
+        logger: globals.logger,
+        platform: globals.platform,
```

Comment:
Should we do `platform: linux` here to ensure that `isWindows` is set to false? (same for other tests above).

Reply:
I updated the PR description to reflect this, but I believe this was erroneous before. We run these tests on Windows and therefore we should always check the platform.